build.gradle

```diff
@@ -6,12 +6,14 @@ plugins {
     id 'nebula.lint' version '16.8.0'
```

Comment:
Why not use the sonatype snapshot repository? I don't think we should add a second source for our artifacts.